src/libasr/pass/intrinsic_function_registry.h

```diff
@@ -2382,6 +2384,81 @@ namespace Mod {
 
 } // namespace Mod
 
+#define i_BitRshift(n, bits, t) EXPR(ASR::make_IntegerBinOp_t(al, loc,      \
+            n, ASR::binopType::BitRShift, bits, t, nullptr))
+#define i_BitLshift(n, bits, t) EXPR(ASR::make_IntegerBinOp_t(al, loc,      \
+            n, ASR::binopType::BitLShift, bits, t, nullptr))
```

Comment:
Move to asr_builder.h

Reply:
Sorry, my bad, we haven't synced LFortran changes. 
Just move this above where other macros are residing.

Reply:
Done.